mlflow/genai/judges/utils.py

```diff
@@ -83,7 +318,79 @@ def invoke_judge_model(
             error_code=INVALID_PARAMETER_VALUE,
         ) from e
 
-    return feedback
+    return InvokeJudgeModelHelperOutput(
+        feedback=feedback,
+        model_provider=provider,
+        model_name=model_name,
+        request_id=request_id,
+        num_prompt_tokens=num_prompt_tokens,
+        num_completion_tokens=num_completion_tokens,
+    )
+
+
+@record_usage_event(InvokeCustomJudgeModelEvent)
+def invoke_judge_model(
+    model_uri: str, prompt: str, assessment_name: str, num_retries: int = 10
+) -> Feedback:
+    """
+    Invoke the judge model.
+
+    First, try to invoke the judge model via litellm. If litellm is not installed,
+    fallback to native parsing using the AI Gateway adapters.
+
+    Args:
+        model_uri: The model URI.
+        prompt: The prompt to evaluate.
+        assessment_name: The name of the assessment.
+        num_retries: Number of retries on transient failures when using litellm.
+    """
+    from mlflow.metrics.genai.model_utils import _parse_model_uri
+
+    model_provider, model_name = _parse_model_uri(model_uri)
+    in_databricks = _is_in_databricks()
+
+    try:
+        output = _invoke_judge_model(
+            model_uri=model_uri,
+            prompt=prompt,
+            assessment_name=assessment_name,
+            num_retries=num_retries,
+        )
+    except Exception:
+        error_traceback = traceback.format_exc()
+        if in_databricks:
+            try:
+                _record_judge_model_usage_failure_databricks_telemetry(
+                    model_provider=model_provider,
+                    endpoint_name=model_name,
+                    error_code="UNKNOWN",
+                    error_message=error_traceback,
+                )
```

Comment:
```suggestion
        if in_databricks:
            try:
                _record_judge_model_usage_failure_databricks_telemetry(
                    model_provider=model_provider,
                    endpoint_name=model_name,
                    error_code="UNKNOWN",
                    error_message=traceback.format_exc(),
                )
```

to avoid useless `traceback.format_exc` call.